src/ResourceManagement/AppService/WebAppRuntimeStack.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Management.AppService.Fluent
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Defines web app runtime stack on Windows operating system.
+    /// </summary>
+    public class WebAppRuntimeStack
```

Comment:
1. Should use ExpandableEnum? 
2. Since this if for Windows, will there be WebAppRuntimeStack for linux?

Reply:
Linux OS is the class RuntimeStack.cs
https://github.com/Azure/azure-libraries-for-net/blob/master/src/ResourceManagement/AppService/RuntimeStack.cs

This class is done similar to RuntimeStack. It does not use Enum for similar reason. We might later put Version or other property in.

Currently this would used in couple with existing e.g. WithPhpVersion etc.